src/sentry/tagstore/current/models/tagvalue.py

```diff
@@ -0,0 +1,81 @@
+"""
+sentry.tagstore.current.models.tagvalue
+~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2017 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import, print_function
+
+import six
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.api.serializers import Serializer, register
+from sentry.constants import MAX_TAG_VALUE_LENGTH
+from sentry.db.models import (
+    Model, BoundedPositiveIntegerField, GzippedDictField, BaseManager, sane_repr
+)
+
+
+class TagValue(Model):
+    """
+    Stores references to available filters.
+    """
+    __core__ = False
+
+    project_id = BoundedPositiveIntegerField(db_index=True)
+    environment_id = BoundedPositiveIntegerField()
+    key_id = BoundedPositiveIntegerField()
+    value = models.CharField(max_length=MAX_TAG_VALUE_LENGTH)
+    # TODO: do we even use this anymore?
+    data = GzippedDictField(blank=True, null=True)
```

Comment:
I have no idea what we use this column for anymore. Nor is grep very helpful. Love Python.